drivers/hmis/lib/form_data/allegheny/occurrence_point_forms/rental_assistance_end_date.json

```diff
@@ -0,0 +1,14 @@
+{
+  "item": [
+    {
+      "link_id": "rental-assistance-end-date",
+      "type": "DATE",
+      "text": "Rental Assistance End Date",
+      "mapping": {
+        "record_type": "ENROLLMENT",
+        "custom_field_key": "rental_assistance_end_date"
+      },
+      "_comment": "TODO any bounds?"
```

Comment:
Definitely min should be enrollment start date, not sure on max.

Reply:
Oh good catch, I meant to update this. I'll add the min date and ask them about max